### PR TITLE
Upgrade dcos v1 11 1

### PR DESCRIPTION
**Pull request by rimusz**
Bump dc/os to v1.11.1, add k8s HA docs, update makefile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,23 @@ admin_cidr = "0.0.0.0/0"
 
 For more advanced scenarios, please check the [terraform-dcos documentation for Google Cloud](https://github.com/dcos/terraform-dcos/tree/master/gcp).
 
+### Kubernetes configuration
+
+**NOTE:** By default, it will provision a Kubernetes cluster with one (1) worker node, and
+a single instance of every control plane component.
+
+To deploy a **highly-available** cluster with three (3) private and one (1) public workers node update `.deploy/options.json`:
```
Comment (jimmidyson) [inline README.md]: Shame we can't just use resources/option-ha.json somehow?

Comment (rimusz) [reply README.md]: HA options aren't same per cloud provider

Comment (jimmidyson) [reply README.md]: The only difference is for AWS, right? Anyway, we can maybe improve that in future, this is great for now.

**Comment by rimusz** [comment]
ok, all changes done and well tested

**Pull request merged**